ext/agent/helpers/helpers.go

```diff
@@ -39,23 +41,43 @@ func ListContainers(cli *client.Client) ([]types.Container, error) {
 	return filteredContainers, nil
 }
 
-func CollectLogsForAnalysis(containerID string, cli *client.Client, logTimeTail string) (string, error) {
+func CollectLogsForAnalysis(containerID string, cli *client.Client) ([]models.LogEntry, error) {
+	var logEntries []models.LogEntry
 	logs, err := cli.ContainerLogs(context.Background(),
 		containerID,
 		types.ContainerLogsOptions{
-			Since:      logTimeTail,
+			Timestamps: true,
+			Tail:       "8",
```

Comment:
Maybe we should extract it to some constant because for now it is like a magic number